src/styles/blocks/main-content.scss

```diff
@@ -0,0 +1,4 @@
+.main-catalog__item {
```

Comment:
```suggestion
.main-catalog { 
  &__item {
```

Reply:
and check file name, block should be main-content, or file should be main-catalog